draft-ietf-httpbis-cache-digest.md

```diff
@@ -286,6 +287,37 @@ Additionally, User Agents SHOULD NOT send CACHE_DIGEST when in "privacy mode."
 
 --- back
 
+# Encoding the CACHE_DIGEST frame as an HTTP Header
+
+Some HTTP/2 protocol stacks do not provide an interface to inject arbitrary HTTP/2 frames while allowing the application to send additional HTTP request headers.
```

Comment:
I have adjusted the text so that it discusses specifically about Service Workers.

It is Service Workers (that provides a Cache _and_ the Fetch API) that we use to emulate Cache-Digests.